_posts/2019-09-18-#15204.md

```diff
@@ -1,10 +1,11 @@
 ---
-layout: post
-title: "#15204 Add Open External Wallet action (GUI)"
+layout: pr
+title: "#15204 Add Open External Wallet action"
+component: GUI
```

Comment:
nit: in `_posts/2019-05-22-#15450.md` the component is lowercased `gui`, and the component values in the other posts are all lowercased, may as well do the same here?

Reply:
Alternatively, consider "GUI" an exception since it is an acronym, keep this one as-is and change the one in `_posts/2019-05-22-#15450.md` to all caps too.